gaitmap/_event_detection_common/_event_detection_mixin.py

```diff
@@ -148,15 +151,15 @@ def _detect_single_dataset(
 
         # convert to min_vel event list
         min_vel_event_list, _ = _segmented_stride_list_to_min_vel_single_sensor(
```

Comment:
Would suggest renaming as the func can now handle more input types besides segmented_stride_list